submissions/bilkisuTace/database/init.sql

```diff
@@ -0,0 +1,96 @@
+-- ----------------------------
+-- Table structure for customers
+-- ----------------------------
+CREATE TABLE customers (
+  customer_id int(11) NOT NULL AUTO_INCREMENT,
+  first_name varchar(20) NOT NULL,
+  last_name varchar(20) NOT NULL,
+  email varchar(20) NOT NULL,
+  password varchar(20) NOT NULL,
+  PRIMARY KEY (customer_id)
+)
+
+ -- ----------------------------
+-- Table structure for addresses
+-- ----------------------------
+CREATE TABLE addresses (
+  customer_id int(11) NOT NULL,
```

Comment:
`customer_id` can't be the `PRIMARY KEY` of this table and also a `FOREIGN KEY`. Kindly check what exactly you are doing